test/suite/test_compact07.py

```diff
@@ -176,6 +176,16 @@ def test_compact07(self):
         # Check that foreground compaction has done some work on the small table.
         self.assertGreater(self.get_pages_rewritten(uri_small), 0)
 
+        # Drop the tables and wait for sometime for them to be removed from the background 
```

Comment:
Can we verify it somehow instead of waiting for some time? We have been having timing issues with compaction tests so I would like to make sure.

Reply:
I agree, we could expose the number of files in the list as a connection statistic.

Reply:
Added here https://github.com/wiredtiger/wiredtiger/pull/9541/commits/bc23e3a1385951c7f233cd0e5a2b55b70266edc0.